### ISSUE TITLE
RFE Socket: Concurrent sessions per source

### ISSUE THREAD

**Issue by ssahani** [systemd/systemd#1939]
There is now way to limit number of sessions per source IP in RHEL 7.1 or upstream .
That is number of concurrent connections allowed per IP.

This is similar to xinetd per_souce conf 

```
per_source — Defines the maximum number of instances for a service per source IP address. It accepts only integers as an argument and can be used in both xinetd.conf and in the service-specific configuration files in the xinetd.d/ directory.
```

What is the scope in socket units for this ?

**Comment by poettering** [comment]
We do have MaxConenctions= for this. I'd be open to adding a MaxConnectionsPerSource= setting that implements this.

**Comment by poettering** [comment]
Implemented by #3607. Closing.
